homeassistant/components/freedompro/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Freedompro",
+  "config": {
+    "step": {
+      "user": {
```

Comment:
`title` and `description` are missing in config step user.

Generate en.json with `python3 -m script.translations develop` to test strings.json.
https://developers.home-assistant.io/docs/internationalization/core